shared/metrics/src/main/java/io/pravega/shared/metrics/DynamicLoggerImpl.java

```diff
@@ -95,6 +95,24 @@ public Counter call() throws Exception {
     }
 
     @Override
+    public long getCounterValue(String name) {
+        Exceptions.checkNotNullOrEmpty(name, "name");
+        String counterName = name + ".Counter";
+        try {
+            Counter counter = countersCache.get(counterName, new Callable<Counter>() {
+                @Override
+                public Counter call() throws Exception {
+                    return underlying.createCounter(counterName);
```

Comment:
Yes, it seems correct. The get call's signature `get(K key, Callable<? extends V> valueLoader)` expects callable. https://google.github.io/guava/releases/15.0/api/docs/com/google/common/cache/Cache.html#get(K, java.util.concurrent.Callable)